rockcraft.yaml

```diff
@@ -2,7 +2,7 @@
 # See LICENSE file for licensing details.
 
 name: github-runner-webhook-router
-base: ubuntu@22.04
+base: ubuntu@24.04
```

Comment:
@javierdelapuente  upgraded to noble, this should fix the setuptools CVE trivy detected.